### PR TITLE
Add p99.9 in benchmark data fetching script and dashboard

### PR DESCRIPTION
**Pull request by carolynhu**
https://github.com/istio/istio/issues/26958

### CODE REVIEW

File: perf_dashboard/benchmarks/views.py [hunk 1]
```diff
@@ -868,21 +928,29 @@ def get_latency_vs_conn_y_series(df, telemetry_mode, quantiles):
...
 def get_latency_vs_qps_y_series(df, telemetry_mode, quantiles):
     y_series_data = []
     for qps in [10, 100, 500, 1000, 2000, 3000]:
         data = df.query('ActualQPS == @qps and NumThreads == 16 and Labels.str.endswith(@telemetry_mode)')
-        if not data[quantiles].head().empty:
-            y_series_data.append(data[quantiles].head(1).values[0] / 1000)
-        else:
+        try:
+            data[quantiles].head().empty
+        except KeyError as e:
```
Comment (richardwxn) [inline perf_dashboard/benchmarks/views.py:948]: probably better to just use get instead of doing try catch to avoid keyerror

Comment (carolynhu) [reply perf_dashboard/benchmarks/views.py:948]: yea, can do that, but doesn't save any # of lines. Will update later

**Review by richardwxn — APPROVED** [review]

**Comment by richardwxn** [comment]
better to wait for presubmit test

**Comment by carolynhu** [comment]
presubmit job is blocked by boskos, we can temporarily ignore this

**Pull request merged**